migrations/versions/a41fc890a5a8_.py

```diff
@@ -0,0 +1,58 @@
+"""empty message
+
+Revision ID: a41fc890a5a8
```

Comment:
Was the application deployed to stage environment ? If it was only in QA + CI , we can reset those databases without problems, but for other environments, we need to create a proper migration which will either remove old `performance_profile` table or alter it.

Reply:
Actually, there are no new changes in `performance_profile` table. So, should this [file](https://github.com/RedHatInsights/ros-backend/pull/29/files/b46fd995a313a15c6015baf31ed7a0945290f9d9#diff-f5e6df0d16ec0fcbf5c876f937ee6b8222043593b875177c50dcd513dc480af2) remain as is and a new migration only for new entities added?

Reply:
Yes, in that case the old migration should remain. We need to have an unbroken chain of modifications to the database that will result in final schema.

Reply:
done.